tests/testthat/test-update_r_packages.R

```diff
@@ -0,0 +1,11 @@
+test_that("updates a file as epected", {
```

Comment:
@jacobvjk, play with this test code to try figure out how it works. 
You can run it interactively and also via `devtools::test()` (or control + shift + T)